src/pcp/js/index.js

```diff
@@ -0,0 +1,490 @@
+/*
```

Comment:
Factor this file a bit better. For example it looks like it could contain "header", "splash", "clientChannel" or so. 
Also it would be helpful if the files in this module were clearly separated according to whether they were attached to a renderer or main process.
The comment contains a few stray CRs, looks like it entered a faulty file conversion/paste from DOS format